code/src/assets/styles/shared.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+const ButtonText = styled(Link)`
+  color: #4aa46f;
+  font-size: 18px;
+  font-weight: 500;
+`;
+const ButtonContainer = styled.button`
+  border: 1px solid #4aa46f;
+  border-radius: calc(var(--spacing) * 2);
+  padding: calc(var(--spacing) * 2) calc(var(--spacing) * 4);
+  transition: all 0.5s ease-in-out;
+  display: grid;
+  justify-self: center;
+  &:hover {
+    background-color: #4aa46f;
+  }
+  &:hover ${ButtonText} {
+    color: #fff;
+  }
+`;
+
+const ButtonsContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin-top: calc(var(--spacing) * 16);
```

Comment:
ahh love your use of grid <3 Not sure what this calc(var--spacing) does. But it looks so neat!  Will google this later

Reply:
The calc() CSS function lets you perform calculations when specifying CSS property values. In this project I defined a variable as spacing then used it with calc function in different places. It can help us to have more control on design. For example if we want to increase or decrease the spaces in whole project it needs just be changed in one place (where we define spacing variable).
You can read more about calc here: https://developer.mozilla.org/en-US/docs/Web/CSS/calc()